refarch-frontend/src/components/TheSnackbar.vue

```diff
@@ -43,7 +42,7 @@ watch(
   () => snackbarStore.level,
   () => {
     color.value = snackbarStore.level;
-    if (color.value === "error") {
+    if (color.value === SNACKBAR_COLORS.ERROR) {
       timeout.value = 0;
     } else {
       timeout.value = defaultTimeout;
```

Comment:
You missed replacing `defaultTimeout` here.
```suggestion
      timeout.value = SNACKBAR_DEFAULT_TIMEOUT;
```
Please check if other places are affected as well.